src/Provider/ApiServiceProvider.php

```diff
@@ -65,7 +65,7 @@ public function register()
             'Dingo\Api\Console\Command\Docs',
         ]);
 
-        if (class_exists('Illuminate\Foundation\Application', false)) {
+        if (class_exists('Illuminate\Contracts\Foundation\Application', false)) {
```

Comment:
> That is why the author of this plugin is checking to see if that class exists (meaning Laravel is installed).

The author was correct to check for the concrete class `Illuminate\Foundation\Application` which means Laravel is installed. You're checking for `Illuminate\Contracts\Foundation\Application` which means `illuminate/contracts` is installed!.